crates/rooch-key/src/keypair.rs

```diff
@@ -28,8 +28,8 @@ use strum_macros::EnumString;
 #[cfg_attr(any(test, feature = "fuzzing"), derive(Arbitrary))]
 #[strum(serialize_all = "lowercase")]
 pub enum KeyPairType {
-    RoochKeyPairType,
-    EthereumKeyPairType,
+    Rooch,
+    Ethereum,
 }
```

Comment:
Make this rename to another PR, and only do the rename.

Reply:
I've moved it back.